doc/source/development.rst

```diff
@@ -99,6 +99,47 @@ directory will take effect without reinstalling the package.
 
 .. warning:: if you run ``python setup.py install``, files will be copied from the Ray directory to a directory of Python packages (``/lib/python3.6/site-packages/ray``). This means that changes you make to files in the Ray directory will not have any effect.
 
+Building Ray on Windows (full)
+------------------------------
+
+**Requirements**
+
+- Microsoft Visual Studio 2019 (or Microsoft Build Tools 2019)
+- JDK 15
+- Miniconda 3
+- git for Windows, version 2.31.1 or later
```

Comment:
Also point out it should be installed into `C:\Program Files\Git`, see instructions below

Reply:
Also it would be good to point out explicitly that this won't work with an older git here

Reply:
The requirement already states that git version must be 3.21.1 or later.